### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by SteffenJw**
changed sigma to log(sigma) and removed leftover mentions of laplace

### CODE REVIEW

**Review by cklyne — COMMENTED** [review]
This isn't what's implemented.
The loss function assumes that the prediction is the std, not the $\log(std)$.
I used the [pytorch implementation](https://pytorch.org/docs/stable/generated/torch.nn.GaussianNLLLoss.html) as a refference, but I replaced the var with the std to avoid issues with logs of negative values.

**Comment by SteffenJw** [comment]
okay nevermind, sorry, I missed when you changed that.

**Review by cklyne — APPROVED** [review]
lgtm
